src/Session.cpp

```diff
@@ -167,7 +171,7 @@ void Session::fillContentFromJson(const std::string &configFilePath) {
             for(uint j = 0; j < episodesNum; j++)
             {
                 int nextEpisodeId = currentId+1;
-                if(j == seriesDesc["seasons"][i].size()-1 && i==seriesDesc["seasons"].size()-1)
+                if(j == episodesNum-1 && i==seriesDesc["seasons"].size()-1)
```

Comment:
is the second size ok? why?

Reply:
because the second is an array and array sizes work as expected.
The problem was that numbers are treated as object of size 1 (as in 1 number)

Reply:
Perfect
Write it in the code